### DISCUSSION TITLE
Unable to build tinyuf2 bootload

### DISCUSSION THREAD

**Discussion by mariopesch** [adafruit/tinyuf2#343] [Q&A]
Hi I'm trying to build the tinyuf2 bootloader for a new esp32s2 based board. Installed everything and first tried to compile one of the existing boards but I always ran into the following error:
```
Directory specified in EXTRA_COMPONENT_DIRS doesn't exist: /Users/x/Documents/Github/tinyuf2/ports/espressif/bootloader_components 
```
it seems like the CMakeList.txt file was changed in the last 2 weeks: https://github.com/adafruit/tinyuf2/commit/c704e12e1ab70c5a746a4d7f7a397604e2c8a387

Did I miss something to install?

Environment:
Python: 3 .9.6
ESP-IDF: v5.1-dirty
Mac OS 13.4.1

**Comment by mariopesch** [comment]
@hathach can you help? Or anyone else?

**Reply by hathach** [reply]
Please use the idf version that is included as a submodule. I believe it is v4.3 or v4.4. IDF v5 includes some breaking changes that we haven't migrated to yet
